dev/com.ibm.ws.javaee.ddmodel/src/com/ibm/ws/javaee/ddmodel/common/XSDTokenType.java

```diff
@@ -45,6 +45,20 @@ public List<String> getList() {
             }
             return stringList;
         }
+
+        public String[] getArray() {
+            String[] stringArray = new String[list.size()];
+            for (int i = 0; i < list.size(); i++) {
+                stringArray[i] = list.get(i).getValue();
+            }
+            return stringArray;
+        }
+
+        public static final String[] EMPTY_ARRAY = new String[] {};
+
+        public static String[] getEmptyArray() {
+            return EMPTY_ARRAY;
+        }
```

Comment:
Optional review comment - You could remove this method and just directly access the constant, which would make it more clear that you are getting the same instance every time.